src/runtime/composables/useDirectusItems.ts

```diff
@@ -49,7 +37,7 @@ export function useDirectusItems<TSchema extends Record<string, any>> () {
     options?: DirectusItemRequestOptions
   ) => {
     const { data, pending, error, refresh } = await useAsyncData(
-      // TODO: check status for https://github.com/nuxt/nuxt/issues/23000
+      // TODO: Following https://github.com/nuxt/nuxt/issues/23000 we could customize the random key via `optimization.keyedComposables`
```

Comment:
We actually can't use `keyedcomposables` because its a nuxt configuration, we are not extending with this module. We should create a random key ourself for that.

Reply:
I had that impression and only now I checked docs.

Currently I use the collection name for `getItems` and `getSingletonItem` as defaults (since they already must be different on Directus side), while `getItemById` is `COLLECTION_ID`.

Since the key appears to be applied at build time, a similar approach to the original one (using filename and line number as seeds). But not only I have no idea on how to do this (just lack of experience I guess), nor I would like to risk to generate slightly different keys per environment build/execution.

Another parameter I was thinking about using in conjunction with the collections name was a short hash based on the query (if used), but what tools should I use for this?